setup.cfg

```
@@ -14,6 +14,7 @@ zip_safe = False
 python_requires = >=3.6
 packages = find:
 install_requires =
+    astropy==4.0.0
```

I would go with
```suggestion
    astropy!=4.0.1
```
because then when 4.0.2 comes out it will pick it up.